packages/ember-glimmer/lib/utils/bindings.ts

```diff
@@ -50,19 +51,24 @@ export function wrapComponentClassAttribute(hash: any[]) {
   let index = keys.indexOf('class');
 
   if (index !== -1) {
-    let [ type ] = values[index];
+    let [ type ] = values[index] as WireFormat.Expressions.TupleExpression | string;
 
     if (type === Ops.Get || type === Ops.MaybeLocal) {
-      let getExp = values[index];
+      let getExp = expectTupleExpression(values[index]);
```

Comment:
Would it be better to use `@glimmer/util`‘s `expect`?

Reply:
`expect` asserts truthiness, but I need to assert arrayiness.

Reply:
Gotcha, thank you!